packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -124,12 +124,19 @@ export interface ICluster extends IResource, ec2.IConnectable {
   readonly kubectlLambdaRole?: iam.IRole;
 
   /**
-   * An AWS Lambda layer that includes `kubectl`, `helm` and the `aws` CLI.
+   * An AWS Lambda layer that includes `kubectl` and `helm`
    *
-   * If not defined, a default layer will be used.
+   * If not defined, a default layer will be used containing Kubectl 1.20 and Helm 3.8
    */
   readonly kubectlLayer?: lambda.ILayerVersion;
 
+  /**
+   * An AWS Lambda layer that contains the `aws` CLI.
+   *
+   * If not defined, a default layer will be used containing the AWS CLI 1.x.
```

Comment:
For completeness?

```suggestion
   * If not defined, a default layer will be used containing the AWS CLI 1.25.70.
```

Reply:
It will change every day. Is that really worthwhile?

Reply:
Will it? I thought these were manually bumped like the kubectl and helm versions, if not, forget it 👍